tests/status_response/test_java.py

```diff
@@ -99,3 +112,431 @@ class TestJavaStatusVersion(BaseStatusResponseTest):
     @fixture(scope="class")
     def build(self):
         return JavaStatusVersion.build({"name": "1.8-pre1", "protocol": 44})
+
+
+@BaseStatusResponseTest.construct
+class TestJavaStatusVersion(BaseStatusResponseTest):
+    EXPECTED_VALUES = [
+        ("fml_network_version", 3),
+        (
+            "channels",
+            [
+                {
+                    "res": "cyclopscore:channel_main",
+                    "version": "1.0.0",
+                    "required": True,
+                },
+                {
+                    "res": "supermartijn642configlib:sync_configs",
+                    "version": "1",
+                    "required": False,
+                },
+                {
+                    "res": "alexsmobs:main_channel",
+                    "version": "1",
+                    "required": False,
+                },
+                {
+                    "res": "sophisticatedcore:channel",
+                    "version": "1",
+                    "required": False,
+                },
+                {
+                    "res": "rftoolsbase:rftoolsbase",
+                    "version": "1.0",
+                    "required": True,
+                },
+                {
+                    "res": "irongenerators:irongenerators",
+                    "version": "1",
+                    "required": False,
+                },
+                {
+                    "res": "xaeroworldmap:main",
+                    "version": "1.0",
+                    "required": True,
+                },
```

Comment:
As I mentioned in the review I just submitted, I think this data is just needlessly too long, and it could easily be reduced to just a handful of channels and mods, covering all possible code branches. At that point, the need for something like `faker` goes significantly down for this data.

I don't really like the idea of introducing randomness into the test codebase, as that could for example lead to tests failing because of some unhandled edge-case, which only got produced by faker this one time. This could be especially frustrating for new contributors, who suddenly see the CI fail, when locally the tests run prefectly fine. If such case is found by someone, they can simply report it as an issue, so that we can handle it, and add a test case for it, so it never happens again.